flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/queryConfig.scala

```diff
@@ -37,6 +37,13 @@ class BatchQueryConfig private[table] extends QueryConfig
 class StreamQueryConfig private[table] extends QueryConfig {
 
   /**
+    * The non-windowed groupby aggregate update the calculation result according a configuration of
+    * time interval. By default non-windowed groupby aggregate will update calculation result each
+    * row.
+    */
+  private var unboundedAggregateUpdateInterval: Time = Time.milliseconds(0)
```

Comment:
I think `updateRate` meant update result in a fixed frequency, so we should register a process-time timer. i.e., System clock driven the trigger.  And `updateInterval` meant update result according coming data. so I we should register a event-time timer. i.e, Data driven the trigger.
I think it again, I suggest using `updateInterval`, i.e., we emit result according event-time of data.
What do you think?